docs/src/tutorials/software-management.md

```diff
@@ -0,0 +1,120 @@
+
+# Introduction
+
+This document describes how to install and use the **Software management** feature in thin-edge.io.
+
+> Note: As of now the software management only supports **debian** based distros(Ex: Ubuntu, Debian),
```

Comment:
In that examples I think we need to highlight RaspberryPi OS more than Ubuntu and Debian as that's the primary target.